### PR TITLE
modify api for expand

### PR DESCRIPTION
**Pull request by EvenLjj**
add appendFilePath method and modify api for expand

### CODE REVIEW

**Review by JervyShi — CHANGES_REQUESTED** [review]

File: core/common/src/main/java/com/alipay/sofa/rpc/common/utils/FileUtils.java [hunk 1]
```diff
@@ -238,4 +238,14 @@ public static boolean cleanDirectory(File dir) {
         }
         return dir.delete();
     }
+
+    public static String appendFilePath(String base, String... left) {
```
[the remark this answers is not in the document]
Comment (EvenLjj) [reply core/common/src/main/java/com/alipay/sofa/rpc/common/utils/FileUtils.java]: OK, have modified.

File: core/common/src/main/java/com/alipay/sofa/rpc/common/utils/FileUtils.java [hunk 2]
```diff
@@ -238,4 +238,14 @@ public static boolean cleanDirectory(File dir) {
...
+        StringBuilder sb = new StringBuilder(base);
+        for (String s : left) {
+            sb.append("/");
```
[the remark this answers is not in the document]
Comment (EvenLjj) [reply core/common/src/main/java/com/alipay/sofa/rpc/common/utils/FileUtils.java]: > Use `File.pathSeparator` instead `/` for more compatibility.
OK, have modified, use `File.separatorChar` instead.

[the remark this answers is not in the document]
Comment (EvenLjj) [reply core/common/src/main/java/com/alipay/sofa/rpc/common/utils/FileUtils.java]: OK, have used `File.separatorChar` instead.

**Review by JervyShi — APPROVED** [review]
LGTM

File: core/common/src/test/java/com/alipay/sofa/rpc/common/utils/FileUtilsTest.java [hunk 3]
```diff
@@ -70,4 +70,21 @@ public void cleanDirectory() throws Exception {
...
+        Assert.assertEquals("baseFilePath" + File.separator + "test" + File.separator + "test2",
+            FileUtils.appendFilePath("baseFilePath", "test", "test2"));
+
+        Assert.assertEquals("baseFilePath" + File.separator + "null" + File.separator + "test",
+            FileUtils.appendFilePath("baseFilePath", null, "test"));
+
+        Assert.assertEquals("baseFilePath" + File.separator + "test" + File.separator + "null",
+            FileUtils.appendFilePath("baseFilePath", "test", null));
+
+        Assert.assertEquals("baseFilePath", FileUtils.appendFilePath("baseFilePath", null));
```
Comment (OrezzerO) [inline core/common/src/test/java/com/alipay/sofa/rpc/common/utils/FileUtilsTest.java:87]: 这里的结果是`baseFilePath/null` 是不是更符合规律? 或者我们将包含 null 值的入参,全部认为是异常场景?

Comment (EvenLjj) [reply core/common/src/test/java/com/alipay/sofa/rpc/common/utils/FileUtilsTest.java]: Ok, remove appendFilePath, can use Paths.get().toString() instead

**Review by OrezzerO — APPROVED** [review]
LGTM

**Pull request merged**